examples/pymc3_howto/lasso_block_update.ipynb

```diff
@@ -4,40 +4,75 @@
    "cell_type": "markdown",
```

Comment:
<p>the formatting doesn't look right. It should be <code>{class}'pymc.sample'</code>  but with backticks instead of apostrophes. <strong>However</strong>, important note, it won't render right! This is an issue with pymc versioned documentation that has been fixed for v4 but the notebooks try to generate links to stable (and this is fixed in latest only)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/279/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>